examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/MCConnectionCallbacks.h

```diff
@@ -0,0 +1,52 @@
+/**
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import "MCCommissionerDeclaration.h"
+
+#import <Foundation/Foundation.h>
+
+#ifndef MCConnectionCallbacks_h
+#define MCConnectionCallbacks_h
+
+/** @brief A container class for User Directed Commissioning (UDC) callbacks. */
+@interface MCConnectionCallbacks : NSObject
+
+/**
+ * @param connectionCompleteCallback is required.
+ * @param commissionerDeclarationCallback is optional.
+ *
```

Comment:
Add a description of what the callbacks are here (you can move some of it from the verifyOrEstablishConnection docs) - similar to comments on previous Android PRs.

Reply:
This is the same description as from ConnectionCallbacks.java in the Android PR. Adding some more detail.